fronts-client/src/strategies/update-collection.ts

```diff
@@ -19,12 +22,19 @@ const collectionToEditionCollection = (
 const updateCollectionStrategy = (
   state: State,
   id: string,
-  collection: CollectionWithNestedArticles
+  collection: CollectionWithNestedArticles,
+  renamingCollection: boolean
```

Comment:
This API is a little surprising to me – I was expecting a separate function! Perhaps it's something to address when we want to improve our update concurrency story more comprehensively.

Reply:
Yep, it's not the most obvious implementation. I was being super lazy. The main reason (though not especially obvious) is to make use of the actions post request. That is, I've kept the payload and response the same shape to reuse the sequence of actions we go through.

But mainly, I was being lazy...

Reply:
@akash1810 I don't have a strong opinion here, and I think this PR is needed enough that we might be best off shipping this now and carding up this refactor as a health task – I'll leave that up to you 😁